internal/templating/templating_engine.go

```
@@ -278,6 +283,29 @@ func (e *Engine) base64Func(value any) (result string, err error) {
 }
 
+// evaluateFunc is a template function that parses and executes an arbitrary string as a template, returning the
+// result. This is useful when a value (like a flag usage string) contains template syntax that needs to be rendered.
+// All functions registered in the engine are available inside the evaluated string. If parsing or execution fails
+// the original text is returned unchanged.
+//
+//	{{ evaluate .Usage . }}
+func (e *Engine) evaluateFunc(text string, data any) (string, error) {
+	clone, err := e.template.Clone()
+	if err != nil {
+		return text, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 293) but it returns nil

(nilerr)